core/src/apps/ton/sign_message.py

```
@@ -0,0 +1,155 @@
+        is_user_friendly=True,
+        is_url_safe=True,
+        is_bounceable=msg.is_bounceable,
+        is_test_only=msg.is_testnet_only,
+    )
+
+    # display
+    ctx.primary_color, ctx.icon_path = lv.color_hex(PRIMARY_COLOR), ICON
+    from trezor.ui.layouts import confirm_final, confirm_unknown_token_transfer
+
+    token = None
+    recipient = Address(msg.destination).to_string(True, True)
```

Ruff (0.8.2)

Boolean positional value in function call

(FBT003)